pkg/utils/metadata.go

```diff
@@ -99,6 +115,75 @@ func GetPortfolio() map[string]float64 {
 	return map[string]float64{}
 }
 
+func GetCurrency() (string, float64) {
+	metadata := Metadata{}
+
+	// Get home directory
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return "USD $", 1.0
+	}
+
+	// Check if metadta file exists
+	configPath := homeDir + "/.cryptgo-data.json"
+	if _, err := os.Stat(configPath); os.IsNotExist(err) {
+		return "USD $", 1.0
+	}
+
+	// Open file
+	configFile, err := os.Open(configPath)
+	if err != nil {
+		return "USD $", 1.0
+	}
+
+	// Read content
+	err = json.NewDecoder(configFile).Decode(&metadata)
+	if err != nil {
+		return "USD $", 1.0
+	}
+
+	currency := metadata.Currency
+	currencyVal := 1.0
+
+	url := "https://api.coincap.io/v2/rates"
+	method := "GET"
```

Comment:
Maybe replace this by using the `/rates/id` endpoint instead? It'll avoid the loop to search for the coin too. You might have to save the currency ID is all though.